index.js

```diff
@@ -60,6 +60,7 @@ function SocketIOFile(socket, options) {
 		var uploadTo = fileInfo.uploadTo || '';
 		var data = fileInfo.data || {};
 		var filename = fileInfo.name;
+		var oFileName = fileInfo.name;
```

Comment:
What about using originalFileName instead of oFileName? It could be confuse other people :)

Reply:
OK, but how about emitObj.oname? 
should we change emitObj.oname to emitObj.originalName ?

Reply:
I'v change oFileName to originalFileName

Reply:
That would be great!